Laptops/trackpad-methods/manual.md

```diff
@@ -1,17 +1,48 @@
 # Fixing Trackpads: Manual
 
-* [Finding the ACPI path](#finding-the-acpi-path)
+* [Checking GPI0](#checking-gpio)
 * [Edits to the sample SSDT](#edits-to-the-sample-ssdt)
-* [Compiling the SSDT](#compiling-the-ssdt)
+* [Enabling Trackpad](#enabling-trackpad)
 * [Wrapping up](#wrapping-up)
 
+## Checking GPI0
 
-## Finding the ACPI path
+The first thing which should be checked is the GPI0 device, which is required for VoodooI2C. The best way to check this is working is to use IORegistryExplorer.
```

Comment:
Might be worth mentioning you myst have macOS installed. Also maybe link to IOReg download:

* https://github.com/khronokernel/IORegistryClone/blob/master/ioreg-302.zip